### ISSUE TITLE
Artefacts using pre-trained weights

### ISSUE THREAD

**Issue by claroche-r** [hjSim/KOALAnet#2]
Hi,

Thanks for the code.

I tried to test the pre-trained models on the Set5 dataset you provided running:
```python main.py --phase 'test' --test_data_path 'Set5/LR/X4/imgs'  --test_label_path 'Set5/HR'  --test_ckpt_pat ./upsampling_network_x4 --factor 4```.

However, the results have grid artefacts like this one:
<img width="400" alt="Screenshot 2022-03-10 at 17 01 44" src="https://user-images.githubusercontent.com/86484344/157702616-86704d55-17cb-4085-95db-0ce0c87f3d25.png">

I am using TensorFlow 1.13 like mentionned in the requirements.

Is there something I missed in the params ?

Thanks,
Charles

**Comment by claroche-r** [comment]
I finally found my error, the code already attributes the name "upsampling_network_x4" so the code was running on init weights...

**Comment by DevashishPrasad** [comment]
Hi @claroche-r ,

I am facing the same issue, how did you exactly solve this issue? Can you please elaborate?

Thanks a lot.

**Comment by DevashishPrasad** [comment]
Thanks a lot for your response! Yes, I figured this issue out and I am sure the weights are getting loaded correctly. However, now I am getting very weird noisy images are output. I have opened another [issue](https://github.com/hjSim/KOALAnet/issues/3) for this.  No matter what changes I do, I still get noisy images.

Did you also face a similar issue?

Thank you so much, I really appreciate the help.
